src/job.rs

```diff
@@ -1,10 +1,24 @@
-extern chrono;
+extern crate chrono;
 
 use chrono::prelude::*;
 
-mod job {
-    struct Job<T> {
-        activity: Fn<T>,
-        deadline: DateTime,
+#[derive(Debug)]
+struct Activity {
+    command: String,
+    args: Vec<String>,
```

Comment:
I assume it may be better here to leave this as `Vec` for the sake of flexibility, but from a performance perspective, if we know the argument list won't grow or shrink based after `Job` creation, it may make sense to simply make this an array of `str` refs.